source/winKernel.py

```diff
@@ -55,6 +56,58 @@ def CreateFile(fileName,desiredAccess,shareMode,securityAttributes,creationDispo
 	return res
 
 
+def createWaitableTimer(securityAttributes=None, manualReset=False, name=None):

```

Comment:
I think github is not showing this right... but it looks like CreateWaitableTimer and SetWaitableTimer are being added here, yet they already exist in master I think.